tests/test_distribution.py

```diff
@@ -22,10 +23,12 @@ def test_registry_service(
     CMD ["echo", "container from my local registry"]
     """
 
-    out = host.check_output(
-        f"curl -sb -H 'Accept: application/json' http://localhost:{host_port}/v2/_catalog"
+    out = json.loads(
+        host.check_output(
+            f"curl -sb -H 'Accept: application/json' http://localhost:{host_port}/v2/_catalog"
+        )
     )
-    assert out == '{"repositories":[]}'
+    assert len(out["repositories"]) == 0
```

Comment:
you're no longer checking whether there is no other output. the way to address the feedback from @dcermak without breaking existing test would have been something like

```python
assert str(out) == '{"repositories":[]}'
```

Reply:
The initial idea was to just check the content of the repository. The previous code was just a shortcut, and since there were no complaints it has survived there till now.

Now I am wondering whether we want to test the complete response from the registry, or we just value that it has started with empty volume and later on it contains only the image we have pushed there.

Reply:
I was thinking more of 
```suggestion
    assert out == {"repositories": []}
```

Reply:
Alright, I will update it accordingly

Reply:
I have managed to make it work with Dirk's suggestion. Thanks